### PR TITLE
Style tables

### PR DESCRIPTION
**Pull request by MiZiet**
If applied these changes will add new styles for table headers (moved table name and added missing column headers).

### CODE REVIEW

**Review by ekowalsk — APPROVED** [review]

File: packages/page-accounts/src/Accounts/index.tsx [hunk 1]
```diff
@@ -106,7 +106,8 @@ function Overview ({ className = '', onStatusChange }: Props): React.ReactElemen
   }, [accountsWithInfo]);
 
   const header = useRef([
-    [t('accounts'), 'start', 3],
+    [], [],
```
Comment (krzysztof-jelski) [inline packages/page-accounts/src/Accounts/index.tsx:109]:
```suggestion
    [t('favorite')], [],
```
Table with header looks really odd when there's no name for the first column.

File: packages/page-staking/src/Actions/index.tsx [hunk 2]
```diff
@@ -116,7 +116,8 @@ function Actions ({ className = '', isInElection, ownStashes, targets }: Props):
   const [typeIndex, setTypeIndex] = useState(0);
 
   const headerRef = useRef([
-    [t('stashes'), 'start', 2],
+    [],
+    [t('account name'), 'start'],
```
Comment (krzysztof-jelski) [inline packages/page-staking/src/Actions/index.tsx:120]:
```suggestion
    [t('stash'), 'start'],
```

File: packages/page-staking/src/Payouts/index.tsx [hunk 3]
```diff
@@ -184,15 +184,15 @@ function Payouts ({ className = '', isInElection, ownValidators }: Props): React
   );
 
   const headerStashes = useMemo(() => [
-    [myStashesIndex ? t('payout/stash') : t('overall/validator'), 'start', 2],
+    [t('account name'), 'start', 2],
```
Comment (krzysztof-jelski) [inline packages/page-staking/src/Payouts/index.tsx:187]:
```suggestion
    [t('stash'), 'start', 2],
```
(though perhaps it's obvious enough as is)

File: packages/page-staking/src/Payouts/index.tsx [hunk 4]
```diff
@@ -184,15 +184,15 @@ function Payouts ({ className = '', isInElection, ownValidators }: Props): React
...
     [t('eras'), 'start'],
     [myStashesIndex ? t('own') : t('total')],
     [('remaining')],
     [undefined, undefined, 3]
   ], [myStashesIndex, t]);
 
   const headerValidatorsRef = useRef([
-    [t('payout/validator'), 'start', 2],
+    [t('account name'), 'start', 2],
```
Comment (krzysztof-jelski) [inline packages/page-staking/src/Payouts/index.tsx:195]:
```suggestion
    [t('validator'), 'start', 2],
```
(though perhaps it's obvious enough as is)

File: packages/page-settings/src/Metadata/NetworkSpecs.tsx [hunk 5]
```diff
@@ -106,9 +102,8 @@ function NetworkSpecs ({ chainInfo, className }: Props): React.ReactElement<Prop
     <Table
       className={className}
       empty={t<string>('No open tips')}
```
Comment (krzysztof-jelski) [inline packages/page-settings/src/Metadata/NetworkSpecs.tsx:104]: The empty string looks copy-pasted from a different table :-)

Comment (MiZiet) [reply packages/page-settings/src/Metadata/NetworkSpecs.tsx]: Changed to `No specification`

File: packages/page-staking/src/Payouts/index.tsx (hunk 4 above)
Comment (MiZiet) [reply packages/page-staking/src/Payouts/index.tsx]: ✅

File: packages/page-staking/src/Payouts/index.tsx (hunk 3 above)
Comment (MiZiet) [reply packages/page-staking/src/Payouts/index.tsx]: ✅

File: packages/page-staking/src/Actions/index.tsx (hunk 2 above)
Comment (MiZiet) [reply packages/page-staking/src/Actions/index.tsx]: ✅

File: packages/page-accounts/src/Accounts/index.tsx (hunk 1 above)
Comment (MiZiet) [reply packages/page-accounts/src/Accounts/index.tsx]: ✅

**Review by ekowalsk — APPROVED** [review]

**Comment by krzysztof-jelski** [comment]
Hey @jacogr , would you like us to implement one of the designs proposed above?

**Comment by jacogr** [comment]
Closing.

**Pull request closed without merging**